packages/data-store/src/__tests__/pd-manager.entities.test.ts

```diff
@@ -50,11 +49,11 @@ describe('PresentationDefinitionItemEntity tests', (): void => {
   })
 
   it('should update PresentationDefinitionItemEntity dcqlPayload', async (): Promise<void> => {
-    const repository = dbConnection.getRepository(PresentationDefinitionItemEntity)
-    const entity = new PresentationDefinitionItemEntity()
-    entity.definitionId = 'ajax-club'
+    const repository = dbConnection.getRepository(DcqlQueryItemEntity)
+    const entity = new DcqlQueryItemEntity()
+    entity.queryId = 'ajax-club'
     entity.version = '1.0'
-    entity.dcqlPayload = JSON.stringify(SAMPLE_DCQL_QUERY_PAYLOAD.dcqlQuery)
+    entity.dcqlPayload = JSON.stringify(SAMPLE_DCQL_QUERY_IMPORT.dcqlQuery)
```

Comment:
this can now just be called "query"? as everything is dcql anyways?

Reply:
It's closer to the ImportDcqlQueryItem it loads.
(And it's in test.ts who cares)